app/views/devise/mailer/reset_password_instructions.html.slim

```diff
@@ -0,0 +1,15 @@
+div style='background-color: #cfdfe4; border-radius: 20px; width: 75%; margin: 0 auto;'
+  h1 style='border-radius: 20px; text-align: center; font-weight: bold; height: 60px; padding-top: 30px;'
+    | Hello 
+    = @resource.full_name
+    | !
+  hr style='width: 90%; text-align: center;'
+  p style='margin: 20px;'
+    p style='width: 90%; margin: 5px 5%; font-size: 20px;'
+      | Someone has requested a link to change your password. You can do this through the link below.
+    p style='width: 90%; margin: 5px 5%; font-size: 20px;'
+      = link_to 'Change my password', edit_password_url(@resource, reset_password_token: @token)
+    p style='width: 90%; margin: 5px 5%; font-size: 20px;'
+      | If you didn't request this, please ignore this email.
+    p style='width: 90%; margin: 5px 5%; font-size: 18px; padding-bottom: 30px'
+      | Your password won't change until you access the link above and create a new one.
```

Comment:
Use styles in scss files

Reply:
I'll be happy to use, but bootstrap and scss can't working

Reply:
You right, only inline styles work for email html files